dashboard/app/controllers/pd/professional_learning_controller.rb

```diff
@@ -29,6 +30,46 @@ def index
     }.compact
   end
 
+  # GET professional-learning/facilitator/computer-science-a
+  def csa
+    @course_name = Pd::Workshop::COURSE_CSA
+    if current_user&.can_view_all_facilitator_landing_pages? || current_user&.courses_as_facilitator&.exists?(course: @course_name)
```

Comment:
nit: is this worth breaking out into a helper method? It looks like it's the same for all of these routes?